samples/drivers/adc/adc_dt/boards/beagleconnect_freedom.overlay

```diff
@@ -0,0 +1,74 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * Copyright (c) 2024 Ayush Singh <ayush@beagleboard.org>
+ */
+
+/ {
+	zephyr,user {
+		/* Reading mulitple pins not possible */
+		io-channels = <&adc0 0x09>;
+		io-channel-names = "MB1_AN";
```

Comment:
Thanks! Perfect! Putting the details (commented out) for MB2_AN might help people check the other AN pin quickly.

Reply:
Done.